07-routing-gossip.md

```diff
@@ -425,7 +425,7 @@ fields in the `channel_update` message:
 Note that the `htlc_maximum_msat` field is static in the current
 protocol over the life of the channel: it is *not* designed to be
 indicative of real-time channel capacity in each direction, which
-would be both a massive data leak and uselessly spam the network (it
+would be both a massive data leak and uselessly spam in the network (it
```

Comment:
NACK, `spam the network` is correct here

Reply:
From this sentence, `which would be both a massive data leak and uselessly spam the network`.
Shouldn't it mean **both** a massive data leak and useless spam in the network?
Are you suggesting the `spam` here is a verb rather than a noun?